src/software/ai/hl/stp/play/defense/defense_play_fsm.cpp

```diff
@@ -75,6 +76,11 @@ void DefensePlayFSM::updateCreaseAndPassDefenders(
     unsigned int max_num_crease_defenders =
         defender_assignment_config.max_num_crease_defenders();
 
+    // Get primary threat position for calculating pass defender positions
+    Point primary_threat_position = enemy_threats.empty()
+                                        ? event.common.world_ptr->ball().position()
+                                        : enemy_threats.front().robot.position();
+
```

Comment:
nit: Our style guide says to avoid ternaries and just use an if/else. I'm personally not opposed, but we should follow our style guide for consistency.